doc/modules/cassandra/pages/cql/functions.adoc

```diff
@@ -238,6 +238,29 @@ For every xref:cql/types.adoc#native-types[type] supported by CQL, the function
 Conversely, the function `blobAsType` takes a 64-bit `blob` argument and converts it to a `bigint` value. 
 For example, `bigintAsBlob(3)` returns `0x0000000000000003` and `blobAsBigint(0x0000000000000003)` returns `3`.
 
+===== Collection functions
```

Comment:
We should mention here the overflow an other comments/problems mentioned in the ticket imo